extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.css

```diff
@@ -3,21 +3,37 @@ amp-story-shopping-attachment {
   width: 100% !important;
   font-family: 'Poppins', sans-serif !important;
   font-size: calc(var(--story-page-vmin) * 3) !important; /* override with ems for pixel perfect layout */
-  --shopping-attachment-horizontal-padding: 1.5em;
+  --i-amphtml-shopping-attachment-horizontal-padding: 1.5em !important;
+  --i-amphtml-shopping-attachment-vertical-padding: 1.2em !important;
+  --i-amphtml-shopping-three-percent-overlay: rgba(0, 0, 0,  .03) !important;
+}
+
+amp-story-shopping-attachment[theme="dark"] {
+  --i-amphtml-shopping-three-percent-overlay: rgba(255, 255, 255, .03) !important;
 }
 
 amp-story-shopping-attachment a {
   text-decoration: none !important;
 }
 
+.i-amphtml-amp-story-shopping {
+  display: none;
+}
+
+.i-amphtml-amp-story-shopping[active] {
+  display: block;
+}
+
+/* PLP (Product Listing Page). */
+
 .i-amphtml-amp-story-shopping-plp {
-  padding: 0 var(--shopping-attachment-horizontal-padding) !important;
+  padding: 0 var(--i-amphtml-shopping-attachment-horizontal-padding) !important;
 }
 
 .i-amphtml-amp-story-shopping-plp-header {
   font-weight: 600 !important;
   font-size: 1.5em !important;
-  margin-bottom: 1.2em !important;
+  margin: var(--i-amphtml-shopping-attachment-vertical-padding) 0 var(--i-amphtml-shopping-attachment-vertical-padding) !important;
```

Comment:
super nitty, but you should be able to only write `margin: var() 0;` without repeating the var twice